Ryujinx.HLE/HOS/Services/Mii/Types/StoreData.cs

```diff
@@ -78,7 +76,7 @@ private ushort CalculateDeviceCrc()
 
         private ReadOnlySpan<byte> AsSpan()
         {
-            return MemoryMarshal.AsBytes(SpanHelpers.CreateReadOnlySpan(in this, 1));
+            return MemoryMarshal.AsBytes(SpanHelpers.CreateReadOnlySpan(ref this, 1));
```

Comment:
Might as well simplify this a bit
```suggestion
            return SpanHelpers.AsReadOnlyByteSpan(in this);
```